### ISSUE TITLE
add a device selector menu

### ISSUE THREAD

**Issue by pq** [flutter/flutter-intellij#137]
(ideally to the top-level toolbar next to the run config chooser)

**Comment by pq** [comment]
Still a WIP and waiting for @stevemessick's goodies to land before we can wire it up but after some fiddling I think it's at least possible....

<img width="338" alt="screen shot 2016-09-22 at 3 16 30 pm" src="https://cloud.githubusercontent.com/assets/67586/18768061/a5d4769c-80d7-11e6-987c-f9fd1d9bc065.png">

👍

**Comment by stevemessick** [comment]
Looks great! Although I agree with a comment you made earlier that we should investigate icons.

**Comment by pq** [comment]
Closing out.  Wiring up will get tackled in another issue.
